### PR TITLE
Enable Display of Built-In Unauthorized

### PR DESCRIPTION
**Pull request by edmondchui**
For issue #664

### CODE REVIEW

File: lib/doorkeeper/rails/helpers.rb [hunk 1]
```diff
@@ -11,11 +11,11 @@ def doorkeeper_authorize!(*scopes)
         end
       end
 
-      def doorkeeper_unauthorized_render_options
+      def doorkeeper_unauthorized_render_options(error)
```
Comment (tute) [inline lib/doorkeeper/rails/helpers.rb]: I'd pick `_error` as a name.

Comment (tute) [inline lib/doorkeeper/rails/helpers.rb]: Also, I'd make it a keyword argument: `(_error:)`.

File: lib/doorkeeper/rails/helpers.rb [hunk 2]
```diff
@@ -11,11 +11,11 @@ def doorkeeper_authorize!(*scopes)
         end
       end
 
-      def doorkeeper_unauthorized_render_options
+      def doorkeeper_unauthorized_render_options(_error:)
```
Comment (tute) [inline lib/doorkeeper/rails/helpers.rb]: I'm sorry, we don't need the underscore here, makes no sense to require such a key. Can you please remove it?

File: lib/doorkeeper/rails/helpers.rb [hunk 3]
```diff
@@ -32,7 +32,7 @@ def doorkeeper_render_error
       end
 
       def doorkeeper_render_error_with(error)
-        options = doorkeeper_render_options || {}
+        options = doorkeeper_render_options(error) || {}
```
Comment (tute) [inline lib/doorkeeper/rails/helpers.rb]: This should read: `error: error`.

File: lib/doorkeeper/rails/helpers.rb [hunk 4]
```diff
@@ -50,11 +50,11 @@ def doorkeeper_error
...
       end
 
-      def doorkeeper_render_options
+      def doorkeeper_render_options(error)
         if doorkeeper_invalid_token_response?
-          doorkeeper_unauthorized_render_options
+          doorkeeper_unauthorized_render_options(error)
         else
-          doorkeeper_forbidden_render_options
+          doorkeeper_forbidden_render_options(error)
```
Comment (tute) [inline lib/doorkeeper/rails/helpers.rb]: These three should read: `error: error`.

File: lib/doorkeeper/rails/helpers.rb (hunk 2 above)
Comment (edmondchui) [inline lib/doorkeeper/rails/helpers.rb]: Oh, I thought the underscore was suggested by houndci and concurred by you in my last commit. Anyway, I'm changing it now.

File: lib/doorkeeper/rails/helpers.rb (hunk 4 above)
Comment (edmondchui) [inline lib/doorkeeper/rails/helpers.rb]: This is the method call line, not the declaration line. Should we still use ```error: error``` as well?

**Comment by tute** [comment]
Thank you!

**Pull request closed without merging**